IronPythonExtension/IronPythonExtension.cs

```diff
@@ -61,26 +62,9 @@ public void Startup(StartupParams sp)
         /// <param name="sp"></param>
         public void Ready(ReadyParams sp)
         {
-            // Searches for DSIronPython engine binary in same folder with extension itself
-            var targetDir = Path.GetDirectoryName(Assembly.GetAssembly(typeof(IronPythonExtension)).Location);
-            var libraryLoader = sp.StartupParams.LibraryLoader;
-            Assembly pythonEvaluatorLib = null;
-            try
-            {
-                pythonEvaluatorLib = Assembly.LoadFrom(Path.Combine(targetDir, PythonEvaluatorAssembly + ".dll"));
-            }
-            catch (Exception ex)
-            {
-                // Most likely the IronPython engine is excluded in this case
-                // but logging the exception message in case for diagnose
-                OnMessageLogged(LogMessage.Info(ex.Message));
-                return;
-            }
-            // Import IronPython Engine into VM, so Python node using IronPython engine could evaluate correctly
-            if (pythonEvaluatorLib != null)
-            {
-                libraryLoader.LoadNodeLibrary(pythonEvaluatorLib);
-            }
+            var extraPath = Path.Combine(new FileInfo(Assembly.GetAssembly(typeof(IronPythonExtension)).Location).Directory.Parent.FullName, "extra");
```

Comment:
Why can't we use `GetExecutingAssembly().Location` here?

Reply:
I think we can, let me test that.